test/unit/genUniqueString.test.ts

```diff
@@ -7,7 +7,7 @@
 
 import { isString } from '@salesforce/ts-types';
 import { expect } from 'chai';
-import { genUniqueString } from '../../lib/genUniqueString';
+import { genUniqueString } from '../../src/genUniqueString';
```

Comment:
This should be `lib` since that's what we ship.

Reply:
I don't think so, since we run tests using ts-node.

Reply:
Well, that goes against "test what you ship" but ok.

Reply:
Well, then all of our repos are doing it wrong. At least I think. i.e I'm pretty sure fancytest uses ts-node too.